src/remove.js

```diff
@@ -0,0 +1,11 @@
+const removeTask = (i) => {
+  let tasks = JSON.parse(localStorage.getItem('datas'));
+  tasks = tasks.filter((task) => task.description !== tasks[i].description);
+  /* eslint-disable no-plusplus */
```

Comment:
- [ ] I'll suggest you do the same thing here and remove the linters disable. :waffle: :+1:  You can replace the `i++` with `i += 1` to resolve this issue here. :+1: